modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
+
+[role="_abstract"]
+When upgrading to a version >= 4.6 from a version < 4.6, you must patch the sensor and admission-control deployments  to set the `POD_NAMESPACE` environment variable.
+
+[NOTE]
+====
+If you are using Kubernetes, use `kubectl` instead of `oc` for the commands listed in this procedure.
+====
+
+.Procedure
+
+. Patch sensor to ensure `POD_NAMESPACE` is set:
```

Comment:
```suggestion
. Patch sensor to ensure `POD_NAMESPACE` is set by running the following command:
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command

Reply:
Done.